src/parser.js

```diff
@@ -0,0 +1,40 @@
+const bent = require('bent');
+const jsdom = require('jsdom');
+
+const request = bent('string');
+const url = 'https://wiki.cdot.senecacollege.ca/wiki/Planet_CDOT_Feed_List';
+const { JSDOM } = jsdom;
+const nameTest = /^name/i;
+const linkTest = /^#/;
+let lines = '';
+let feed = '';
+let feedList = '';
+let tempLine = '';
+
+/**
+* Get feed data from url
+* process it to remove unnecessary information like "name" and square brackets
+*/
+request(url)
```

Comment:
Let's wrap this in a function we can call.  In fact, let's split it into 2 functions.  One function can download the data from the wiki and parse out the `<pre>`; the second function can accept the text inside it and do the processing. This should make it easier to write tests for each.